src/components/layout/tables/AntRow/AntRow.tsx

```diff
@@ -0,0 +1,186 @@
+    }
+    // todo: remove below for v1, this is a legacy format for ant record antStates.
+    if (typeof antState.records['@'] === 'string') {
+      transactionId = antState.records['@'];
+    }
+    setAntDetails({
+      1: name(),
+      2: isMobile ? (
+        <CopyTextButton
+          displayText={`${antId.slice(0, 2)}...${antId.slice(-2)}`}
+          copyText={antId}
+          size={24}
+        />
+      ) : (
+        <CopyTextButton
+          displayText={`${antId.slice(0, 6)}...${antId.slice(-6)}`}
+          copyText={antId}
+          size={24}
+        />
+      ),
+      3: transactionId ? (
+        isMobile ? (
+          <CopyTextButton
+            displayText={`${transactionId.slice(0, 2)}...${transactionId.slice(
+              -2,
+            )}`}
+            copyText={transactionId}
+            size={24}
+          />
+        ) : (
+          <CopyTextButton
+            displayText={`${transactionId.slice(0, 6)}...${transactionId.slice(
+              -6,
+            )}`}
+            copyText={transactionId}
+            size={24}
+          />
+        )
+      ) : (
+        'N/A'
+      ),
+      4: (
+        <span className="text white bold center">
+          {icon()}&nbsp;{!isMobile ? `${confirmations} / 50` : <></>}
+        </span>
+      ),
+      5: <ManageAssetButtons asset={antId} assetType={'ant'} />,
+    });
+
+    return;
+  }
+
+  return (
+    <>
+      <tr
+        className="assets-table-header"
+        style={
+          bgColor
+            ? {
+                backgroundColor: bgColor,
+                color: textColor,
+              }
+            : {}
+        }
+      >
+        {antDetails ? (
+          Object?.values(antDetails).map((value: any, index) => (
+            <td
+              className="assets-table-item"
+              style={textColor ? { color: textColor } : {}}
+              key={index}
+            >
+              {value}
+            </td>
+          ))
+        ) : (
+          <>
+            <td
+              className="assets-table-item"
+              style={textColor ? { color: textColor } : {}}
+            >
+              <Loader size={30} />
+            </td>
+            <td
+              className="assets-table-item"
+              style={textColor ? { color: textColor } : {}}
+            >
+              <Loader size={30} />
+            </td>
+            <td
+              className="assets-table-item"
+              style={textColor ? { color: textColor } : {}}
+            >
+              <Loader size={30} />
+            </td>
+            <td
+              className="assets-table-item"
+              style={textColor ? { color: textColor } : {}}
+            >
+              <Loader size={30} />
+            </td>
+            <td
+              className="assets-table-item"
+              style={textColor ? { color: textColor } : {}}
+            >
+              <Loader size={30} />
```

Comment:
redundant - either set defaults in ant state or use headers to dynamically set all these on load